src/entry.ts

```diff
@@ -134,6 +129,17 @@ function processGenerationOption(session: Session<CodeModel>) {
         ? AzConfiguration.getValue(CodeGenConstants.compatibleLevel)
         : compatibleLevel;
     const isTrack1 = compatibleLevel === CompatibleLevel.Track1;
+    session.message({
+        Channel: Channel.Warning,
```

Comment:
Is it better to use information here?

Reply:
I am wondering whether infomation will show up in non-debug mode. Will check that, and change it back if information can show up.

Reply:
Updated